### PR TITLE
Expose library system name & servicepoint ID in library and for querying

### PR DESCRIPTION
**Pull request by humble-ahitofel**
Exposes library system name and ID in API.
See: https://github.com/libraries-fi/kirkanta/pull/36

### CODE REVIEW

**Comment by humble-ahitofel** [comment]
Escaped the underscores in library_system_id, markdown interpreted them as italic formatting.

File: doc/documentation.md [hunk 1]
```diff
@@ -143,31 +144,34 @@ https://api.kirjastot.fi/v4/library/<id>
...
---              | --
-created.after   | Lower bound for the date the document was created.
-created.before  | Upper bound for the date the document was created.
-modified.after  | Lower bound for document modification date.
-modified.before | Upper bound for document modification date.
---              | --
-with            | Return additional data blocks.
-refs            | Collect specified linked data.
+Name                  | Description
+-----------------     | -----------
```
Comment (ajsalminen) [inline doc/documentation.md]:
```suggestion
--------------------- | -----------
```

**Comment by ajsalminen** [comment]
Can we add a line about the purpose of these to be included in commit message (on both this and kirkanta) for documentation purposes?

**Comment by humble-ahitofel** [comment]
> Can we add a line about the purpose of these to be included in commit message (on both this and kirkanta) for documentation purposes?

Hmm, yes the commit seems to be quite a stub. I'll add some additional comments to both pulls.

**Comment by humble-ahitofel** [comment]
Okay, I added a additional clarifying comment. Will this do?

File: doc/documentation.md [hunk 2]
```diff
@@ -143,31 +144,34 @@ https://api.kirjastot.fi/v4/library/<id>
...
---              | --
-created.after   | Lower bound for the date the document was created.
-created.before  | Upper bound for the date the document was created.
-modified.after  | Lower bound for document modification date.
-modified.before | Upper bound for document modification date.
---              | --
-with            | Return additional data blocks.
-refs            | Collect specified linked data.
+Name                              | Description
+---------------------             | -----------
```
Comment (ajsalminen) [inline doc/documentation.md]: Hmmm, this change seems to have gone missing or I guess the column just got longer again.
```suggestion
--------------------------------- | -----------
```

Comment (humble-ahitofel) [reply doc/documentation.md]: My mistake :sweat_smile:
Doc build is looking good after this.

**Comment by ajsalminen** [comment]
I made some minor changes to documentation.md but didn't generate the html version Should be good to go after that, can you update it?

**Comment by humble-ahitofel** [comment]
> I made some minor changes to documentation.md but didn't generate the html version Should be good to go after that, can you update it?

Yes, I tested it. It looked fine.

**Pull request merged**